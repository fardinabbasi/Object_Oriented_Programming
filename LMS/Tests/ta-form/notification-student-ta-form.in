POST login ? id 0 password UT_account
POST course_offer ? course_id 1 professor_id 810201475 capacity 60 time Saturday:14-16 exam_date 1403/4/3 class_number 5
POST course_offer ? course_id 2 professor_id 810203542 capacity 80 time Sunday:14-16 exam_date 1403/4/4 class_number 9
POST logout ?
POST login ? id 810101551 password hehe
PUT my_courses ? id 1
POST logout ?
POST login ? id 810101456 password blabla
POST connect ? id 810201475
POST logout ?
POST login ? id 810201475 password ry
POST course_post ? id 1 title "next session" message "canceled"
POST post ? title "start" message "hey"
POST ta_form ? course_id 1 message "need TA for AP"
POST logout ?
POST login ? id 810101551 password hehe
GET course_channel ? id 1
GET notification ?
POST logout ?