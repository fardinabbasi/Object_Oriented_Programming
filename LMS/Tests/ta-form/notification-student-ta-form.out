OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
1 Advanced Programming 60 khousravi Saturday:14-16 1403/4/3 5
1 khousravi "next session"
1 Advanced Programming: New Course Post
810203542 rahami: New Course Offering
810201475 khousravi: New Course Offering
OK
